app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx

```diff
@@ -43,13 +54,17 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({
   onSearchPage,
 }) => {
   const navigate = useNavigate();
```

Comment:
rearranged to be easier to read, can't remember if this is the same order you do @Fosol?  or if I got it backwards :P

Reply:
customhooks

reacthooks

Reply:
I always sort by hook dependencies first, then state, then variables, then functions.